File: JavaScript_Courses/JavaScript_Basics/Preparation/01_First_Steps_In_Coding_Exercise/08_Basketball_Equipment.js
function basketballEquipment(input) {
    let yearFee = Number(input[0]);

    let shoes = 0.60 * yearFee;
    let equip = 0.80 * shoes;
    let ball = 0.25 * equip;
    let accessoirs = 0.20 * ball;

    let total = yearFee + shoes + equip + ball + accessoirs;

    console.log(total);
}

// basketballEquipment(["365"]);
// basketballEquipment(["550"]);

File: Python_Courses/Python_Basics/Python_PB_2024_10/01_First_Steps_in_Coding_Exercise/08_Basketball_Equipment.py
# •	Баскетболни кецове – цената им е 40% по-малка от таксата за една година
# •	Баскетболен екип – цената му е 20% по-евтина от тази на кецовете
# •	Баскетболна топка – цената ѝ е 1 / 4 от цената на баскетболния екип
# •	Баскетболни аксесоари – цената им е 1 / 5 от цената на баскетболната топка

annual_fee = int(input())

sneakers = annual_fee * 0.60  # shoes = annual_fee * (1 - 40 / 100)
equip = sneakers * 0.80
ball = equip * 0.25  # ball = equip / 4
accessories = ball * 0.20  # accessories = ball / 5

total_result = annual_fee + sneakers + equip + ball + accessories

print(total_result)
